NAMESPACE

```diff
@@ -30,6 +30,7 @@ export(correlation)
 export(crps_sample)
 export(dispersion)
 export(dss_sample)
+export(get_complete_forecasts)
```

Comment:
Is this a good name? To me it implies that this will return all forecasts that are complete for a forecast unit? I believe this is correct. Am I right?

Reply:
rethinking this, I think the original approach was a terrible idea and we should just use `na.omit`